### PR TITLE
Improve communication to liquid issuance service

### PR DESCRIPTION
**Pull request by eckelj**
added OSC response listener with logging
added Machine NFT issuance for each machine attestation process
added CID and Planetmint Issuer extPublicKey to the issuance process

### CODE REVIEW

File: x/machine/keeper/issue_response.go [hunk 1]
```diff
@@ -0,0 +1,25 @@
+package keeper
+
+import (
+	"github.com/cometbft/cometbft/libs/log"
+	"github.com/hypebeast/go-osc/osc"
+)
+
+func (k Keeper) IssueResponseHandler(logger log.Logger) {
+	addr := "0.0.0.0:8766"
```
Comment (jmastr) [inline x/machine/keeper/issue_response.go:9]: Is it a good idea to hard-code the port like that?

File: x/machine/keeper/msg_server_attest_machine.go [hunk 2]
```diff
@@ -44,16 +50,22 @@ func validateIssuerLiquid(issuerLiquid string) bool {
 	return isValidLiquidKey
 }
 
-func (k msgServer) reissueMachine(machine *types.Machine) error {
+func (k msgServer) issueMachineNFT(machine *types.Machine) error {
```
Comment (jmastr) [inline x/machine/keeper/msg_server_attest_machine.go:53]: we have reissue in our proto:
```
proto/planetmintgo/machine/machine.proto:11:  bool reissue = 4;
```

File: x/machine/keeper/issue_response.go (hunk 1 above)
Comment (eckelj) [reply x/machine/keeper/issue_response.go:9]: it's not a good idea. but it's in any case temporarily. will change it.

File: x/machine/keeper/msg_server_attest_machine.go (hunk 2 above)
Comment (eckelj) [reply x/machine/keeper/msg_server_attest_machine.go:53]: That's different. The reissue property is about the character of the token. the function is not about reissuance but issuing an NFT or a more generic token https://help.blockstream.com/hc/en-us/articles/900005127583-How-do-I-issue-an-asset-on-Liquid-

File: config/config.dev.json [hunk 3]
```diff
@@ -1,4 +1,5 @@
 {
 	"WATCHMEN_ENDPOINT": "lab.r3c.network",
-	"WATCHMEN_PORT": 7401
+	"WATCHMEN_PORT": 7401,
+	"OSC_SERVICE_PORT": 8766
```
Comment (jmastr) [inline config/config.dev.json:4]: The value is now missing in `config.prod.json`

**Review by jmastr — CHANGES_REQUESTED** [review]
Add OSC_SERVICE_PORT to config.prod.json

**Review by jmastr — APPROVED** [review]

**Pull request merged**